README.md

```diff
@@ -1,57 +1,99 @@
 
-This project framework provides the following features:
+We recommend following the chapters in successive order. However, the code samples are self-contained, so feel free to pick samples by topics that you may need at the moment.
 
-* Feature 1
-* Feature 2
-* ...
+> :warning: This is a *work in progress*. Come back frequently to discover more samples.
 
-## Getting Started
+## Prerequisites
 
-### Prerequisites
+- [Node.js](https://nodejs.org/en/download/)
+- [Visual Studio Code](https://code.visualstudio.com/download)
+- [VS Code Azure Tools Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+- A modern web browser -while **MSAL React** supports **IE11**, samples here use **ES6** conventions and will not run on **Internet Explorer**
 
-(ideally very short, if any)
+Please refer to each sample's README for sample-specific prerequisites.
 
-- OS
-- Library version
-- ...
+## Recommendations
 
-### Installation
+- [jwt.ms](https://jwt.ms) for inspecting your tokens
+- [Fiddler](https://www.telerik.com/fiddler) for monitoring your network activity and troubleshooting
+- Follow the [Azure AD Blog](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/bg-p/Identity) to stay up-to-date with the latest developments
 
-(ideally very short)
+Please refer to each sample's README for sample-specific recommendations.
 
-- npm install [package name]
-- mvn install
-- ...
+## Contents
 
-### Quickstart
-(Add steps to get up and running quickly)
+- For **Azure AD**, start the tutorial from [here](./1-Authentication/1-sign-in/README-incremental.md)
+- For **Azure AD B2C**, start the tutorial from [here](./1-Authentication/2-sign-in-b2c/README-incremental.md)
 
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+Alternatively, choose below the sample you want to review.
 
+### Chapter 1: Sign-in users and call Microsoft Graph
 
-## Demo
+|               |               |
+|---------------|---------------|
+| <img src="./1-Authentication/1-sign-in/ReadmeFiles/topology.png" width="200"> | [**Sign-in with Azure AD**](./1-Authentication/1-sign-in) </br> Sign-in your users with the **Azure AD** and learn to work with **ID Tokens**. Learn how **single sign-on (SSO)** works. Learn to secure your apps to operate in **national clouds**. Acquire  **access tokens** and learn how to make an **authorized call** to **Microsoft Graph** API.|
```

Comment:
>1-sign-in [](start = 129, length = 9)

append the path to to readme incremental here

Reply:
This points to standalone sample readme. Incremental readme is right above this (under "contents")